resources/zib/terminology/CodeSystem-RoleCodeNLZorgverlenertypen-uzi-rolcode.xml

```diff
@@ -16,7 +20,7 @@
       <system value="urn:ietf:rfc:3986"/>
       <value value="urn:oid:2.16.840.1.113883.2.4.15.111"/>
    </identifier>
-   <version value="11.4.0"/>
+   <version value="11.7.0"/>
```

Comment:
This version of the CodeSystem deprecates codes and falls outside the scope of ZIBFHIR-67

Reply:
It's the code system that deprecates codes. The resource here only reflects these changes.